tests/lca/imagebasedupgrade/mgmt/upgrade/tests/e2e-upgrade-test.go

```diff
@@ -325,21 +334,60 @@ var _ = Describe(
 			}
 		})
 
-		It("upgrades the connected cluster", reportxml.ID("71362"), func() {
+		It("upgrades the connected cluster to a newer minor version", reportxml.ID("71362"), func() {
 			By("Check if the target cluster is connected")
 			connected, err := cluster.Connected(APIClient)
 
 			if !connected {
 				Skip("Target cluster is disconnected")
 			}
 
+			Expect(err).NotTo(HaveOccurred(), "error checking if the cluster is connected")
+
+			By("Check if seed and target have the same minor version")
+			seedImageClusterVersionXY, err := ClusterVersionXY(MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion)
+			Expect(err).NotTo(HaveOccurred(), "error retrieving the XY portion of the seed cluster version")
+
+			if seedImageClusterVersionXY == originalClusterVersionXY {
+				Skip("XY portion of the OCP version between seed and target clusters is identical")
+			}
+
 			if err != nil {
 				Skip(fmt.Sprintf("Encountered an error while getting cluster connection info: %s", err.Error()))
 			}
 
 			upgrade()
 		})
 
+		It("upgrades the connected cluster to a newer z-stream", reportxml.ID("79176"), func() {
+			By("Check if the target cluster is connected")
+			connected, err := cluster.Connected(APIClient)
+
+			if !connected {
+				Skip("Target cluster is disconnected")
+			}
+
+			Expect(err).NotTo(HaveOccurred(), "error checking if the cluster is connected")
+
+			By("Check if seed and target have the same minor version")
+			seedImageClusterVersionXY, err := ClusterVersionXY(MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion)
+			Expect(err).NotTo(HaveOccurred(), "error retrieving the XY portion of the seed cluster version")
+
+			if seedImageClusterVersionXY != originalClusterVersionXY {
+				Skip("XY portion of the OCP version between seed and target clusters is different")
+			}
+
+			By("Check if seed image and target cluster have different XYZ version prior the upgrade")
+			clusterVersion, err := clusterversion.Pull(APIClient)
+			Expect(err).NotTo(HaveOccurred(), "error pulling clusterversion")
+
+			Expect(MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion).To(Not(
+				Equal(clusterVersion.Object.Status.Desired.Version)),
+				"error: clusterversion matches seedimageversion before IBU")
```

Comment:
Isn't this redundant with like 376?

Reply:
376 is supposed to be for XY and this is supposed to be XYZ check